gremlin-test/src/main/java/org/apache/tinkerpop/gremlin/process/ProcessStandardSuite.java

```diff
@@ -144,6 +145,7 @@ public class ProcessStandardSuite extends AbstractGremlinSuite {
             AddEdgeTest.Traversals.class,
             AddVertexTest.Traversals.class,
             CoalesceTest.Traversals.class,
+            ConcatTest.Traversals.class,
```

Comment:
I believe this should also be added to ProcessComputerSuite, it can likely stay out of the Limited suites though

Reply:
I actually haven't thought much about the suites so I will keep this in mind.